### PR TITLE
Re-enable inflation again for testnet

### PR DESCRIPTION
**Pull request by ryoqun**
#### Problem

Inflation is disabled

#### Summary of Changes

Enable it now that the inflation bug is fixed

Follow-up of: #11072

### CODE REVIEW

File: genesis-programs/src/lib.rs [hunk 1]
```diff
@@ -25,10 +25,13 @@ pub fn get_inflation(operating_mode: OperatingMode, epoch: Epoch) -> Option<Infl
...
             0 => Some(Inflation::new_disabled()),
             // testnet enabled inflation at epoch 44:
-            // https://github.com/solana-labs/solana/blob/d8e885f4259e6c7db420cce513cb34ebf961073d
+            // https://github.com/solana-labs/solana/commit/d8e885f4259e6c7db420cce513cb34ebf961073d
             44 => Some(Inflation::default()),
             // Completely disable inflation prior to ship the inflation fix at epoch 68
             68 => Some(Inflation::new_disabled()),
+            // Enable again after the inflation fix has landed:
+            // https://github.com/solana-labs/solana/commit/7cc2a6801bed29a816ef509cfc26a6f2522e46ff
+            72 => Some(Inflation::default()),
```
Comment (ryoqun) [inline genesis-programs/src/lib.rs:34]:
```
$ solana --url http://testnet.solana.com:8899 epoch-info --confirmed

Slot: 24750701
Epoch: 70
Epoch slot range: [24716256..25148256)
Epoch completed percent: 7.973%
Epoch completed slots: 34445/432000 (397555 remaining)
Epoch completed time: 3h 49m 38s/2days (1day 20h 10m 22s remaining)

```

File: genesis-programs/src/lib.rs [hunk 2]
```diff
@@ -25,10 +25,13 @@ pub fn get_inflation(operating_mode: OperatingMode, epoch: Epoch) -> Option<Infl
             // No inflation at epoch 0
             0 => Some(Inflation::new_disabled()),
             // testnet enabled inflation at epoch 44:
-            // https://github.com/solana-labs/solana/blob/d8e885f4259e6c7db420cce513cb34ebf961073d
```
Comment (ryoqun) [inline genesis-programs/src/lib.rs:28]: Bad url...

**Comment by ryoqun** [comment]
@t-nelson https://github.com/solana-labs/solana/pull/10914 has landed and back-ported successfully, I think it can be enabled on testnet anytime. :) Feel free to merge this for ops' convenient timing!

**Comment by t-nelson** [comment]
@ryoqun OK, thanks!  This LGTM!  I'm going to hold off the merge for a bit as there's an unrelated fix coming down the pipe that I'd like to get into the next release, but may necessitate pushing back the activation epoch

**Pull request merged**